### DISCUSSION TITLE
SDK 1.9 Why BT-634-Procedure and BT-634-Lot are available ?

### DISCUSSION THREAD

**Discussion by luispozoWS** [OP-TED/eForms-SDK#1051] [Q&A] [answered]
Hello,
I would like to know why are BT-634-Procedure and BT-634-Lot available in notice type 16 if they can't be filled. There is a rule that allows them only for "Change" or "Result" notices.

Best regards,
Luis

**Comment by YvesJo** [comment, marked as answer]
Indeed BT-634 is allowed for Competition forms. It wouldn't make sense however to publish a competition notice with a procurement relaunch while without that publication the procurement wouldn't exist. However, after that publication, it may be detected that the procurement has to be relaunched and this could be done with a Change notice which will use exactly the same form as the initial notice.
